.github/workflows/wheels.yml

```diff
@@ -125,3 +127,44 @@ jobs:
         with:
           name: ${{ matrix.python }}-${{ startsWith(matrix.buildplat[1], 'macosx') && 'macosx' || matrix.buildplat[1] }}
           path: ./wheelhouse/*.whl
+
+      - name: Setup Upload Variables
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          if [ "push" == "${{ github.event_name }}" ] && [ "${{ startsWith(github.ref, 'refs/tags/v') }}" ]; then
+            echo push and tag event
+            echo "ANACONDA_ORG=multibuild-wheels-staging" >> $GITHUB_ENV
+            echo "TOKEN=${{ secrets.NUMPY_STAGING_UPLOAD_TOKEN }}" >> $GITHUB_ENV
+            echo "ANACONDA_UPLOAD=true" >> $GITHUB_ENV
+          elif [ "schedule" == "${{ github.event_name }}" ] || [ "workflow_dispatch" == "${{ github.event_name }}" ]; then
+            echo scheduled or dispatched event
+            echo "ANACONDA_ORG=scipy-wheels-nightly" >> $GITHUB_ENV
+            echo "TOKEN=${{ secrets.NUMPY_NIGHTLY_UPLOAD_TOKEN }}" >> $GITHUB_ENV
```

Comment:
@bashtage I used `${{ secrets.NUMPY_TOKEN }}` here like elsewhere in this repo where you used `$NUMPY_TOKEN`. I wonder if it makes a difference?